src/cats/monad/channel.cljc

```diff
@@ -49,19 +40,13 @@
 (def ^{:no-doc true}
   channel-monad
   (reify
-    proto/Functor
-    (fmap [mn f mv]
-      (let [ctx m/*context*
-            channel (chan)]
-        (take! mv (fn [v]
-                    (put! channel
-                          ;; Set double monad for handle properly
-                          ;; monad transformers
-                          (m/with-monad ctx
```

Comment:
Hmm... The either transformer tests were failing with the "calling return/pure/mzero without a context" exception before I removed it, and then passed afterward.

Feel free to pick this up where I left off or discard my changes as you see fit. I have to restrain myself from fun projects and focus on work for a at least a week or so..